src/Uno.Foundation/Rect.cs

```diff
@@ -104,8 +105,18 @@ public Rect(Point point1, Point point2)
 			);
 		}
 
-		public static implicit operator string(Rect rect) =>
-			$"{rect.X},{rect.Y},{rect.Width},{rect.Height}";
+		public static implicit operator string(Rect rect)
```

Comment:
That's a lot of noise for a debug string that is not part of the WinUI API .. and actually the most important part here is missing: `$"{rect.X:F2},{rect.Y:F2},{rect.Width:F2},{rect.Height:F2}"`